src/standalone/Soil/rre.jl

```diff
@@ -168,6 +177,7 @@ Construct the tendency computation function for the explicit terms of the RHS,
 which are horizontal components and source/sink terms.
 """
 function ClimaLSM.make_compute_exp_tendency(model::Soil.RichardsModel)
+    # No update boundars required for this model's explicit tendency currently.
```

Comment:
typo: boundars -> boundaries

Reply:
also, could you clarify that this is because we only have BCs in the vertical and those are stepped implicitly? (at first I thought this was suggesting this model doesn't have any BCs)

Reply:
I updated! note that the follow up PR moving `update_boundary_vars!` outside of the tendency will remove this comment also.